src/main/java/com/mycompany/exercisethree/Calculator.java

```diff
@@ -0,0 +1,82 @@
+package com.mycompany.exercisethree;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ *
+ * @author Cristian Gallego
+ */
+public class Calculator {
+
+    public List<Double> history = new ArrayList<>();
+    public List<Double> listNumbers = new ArrayList<>();
```

Comment:
You can make this as local

Reply:
You mean, create this List in each extended class?